src/Loader.php

```diff
@@ -341,6 +342,14 @@ public static function register_scripts() {
 
 		wp_set_script_translations( 'wc-date', 'woocommerce-admin' );
 
+		wp_register_script(
```

Comment:
lets wrap this in a feature check using `wc_admin_get_feature_config` so the nav isn't loaded when unreleased.

Reply:
This actually appeared to be registered twice and is for the exported nav package, not the new navigation feature.

Removed the extra registration, but can see this might be a point of confusion going forwards with those 2 items so similarly named.